[DOCS-1974] Update note about org-wide viewer role (#2024)
wandb/docs: platform/hosting/iam/scim.mdx

| `value` | string | Yes | Role name (`admin` or `member`) |

<Note>
The organization-scoped `viewer` role is deprecated and can no longer be assigned in the UI. If you use SCIM to assign the `viewer` role to a user:
- They are assigned the `member` role in the organization.
- They are assigned a Models `viewer` seat, instead of a `full` seat. This allows view-only access to Models and full access to Registry. If no Models seats are available, a `Seat limit reached` error is logged and the member is added with no Models access. This can be updated later if a seat is available.
- They are assigned a Weave `viewer` seat, instead of a `full` seat. This allows view-only access to Weave. If no Weave seats are available, a `Seat limit reached` error is logged and the member is added with no Weave access. This can be updated later if a seat is available.

> - For organizations that use **Registry**, the user is automatically assigned the `viewer` role in registries that are visible at the organization level.

Author: ```suggestion
- They are assigned the Registry `viewer` role in registries that are visible at the organization level.
```
Simplified this since they still get that role even if there are not yet any registries, I think. Removed the bold to align with the other bullets.
